tests/baselines/reference/privateNameFieldAccess.js

```diff
@@ -0,0 +1,21 @@
+//// [privateNameFieldAccess.ts]
+class Test {
+    #field: number;
+    method() {
+        console.log(this.#field);
+    }
+}
+

```

Comment:
is there a way to test the es2015 emit? I'd like to see what this looks like with a `class` class

Reply:
//@target: ES2015